<?xml version="1.0" ?>

<launch>

    <!-- Start ROS Bag -->
    <param name="use_sim_time" value="true" />
    <arg name="bag"        default="/home/soslab/test4_highF.bag" />
    <arg name="bag_start"  default="0" /> 
    <node pkg="rosbag" type="play" name="rosbag" output="screen" 
          args="--clock --pause -d 1 -s $(arg bag_start) $(arg bag)" required="true"/>

    <!-- TF -->
    <include file="$(find rov_onboard)/launch/sensors/urdf/urdf_test.launch" />

    <!-- Odometry -->
    <include file="$(find rov_processing)/launch/robot_localization/test_odom.launch" />

    <!-- Visualization -->
    <node name="path_node" pkg="rov_processing" type="pub_path" output="screen" >
        <remap from="odometry_1" to="/rov/processed/robot_localization_odometry"/>
        <remap from="path_1" to="/rov/processed/robot_localization_path"/>
    </node>

</launch>